packages/compose/README.md

```diff
@@ -254,6 +254,10 @@ _Parameters_
 -   _object_ `Object`: Object reference to create an id for.
 -   _prefix_ `string`: Prefix for the unique id.
 
+<a name="useIsomorphicLayoutEffect" href="#useIsomorphicLayoutEffect">#</a> **useIsomorphicLayoutEffect**
+
+Undocumented declaration.
```

Comment:
It would be helpful to add JSDoc that explains its purpose. Now developers have 3 similar hooks to pick from.

Reply:
I took a stab at documentation for it, but I'm honestly not sure if I've covered it completely. Let me know if you think this still needs updates.

Reply:
It looks great. It explains its purpose 👍